tensorflow/tools/ci_build/ci_parameterized_build.sh

```diff
@@ -501,14 +501,21 @@ fi
 
 chmod +x ${TMP_SCRIPT}
 
+# Map TF_BUILD container types to containers we actually have.
+if [[ "${CTYPE}" == "android_nightly" ]]; then
```

Comment:
android_nightly doesn't run any explicit tests, but it does provide more complete build coverage and also exposes all the conceivable pre-built targets users may need.